medusa/tv/episode.py

```diff
@@ -751,8 +751,7 @@ def to_json(self, detailed=True):
     def create_meta_files(self):
         """Create episode metadata files."""
         if not self.show.is_location_valid():
-            logger.warning('{id}: The show dir is missing, unable to create metadata', id=self.show.indexerid),
-
+            logger.warning('{id}: The show dir is missing, unable to create metadata', id=self.show.indexerid)
```

Comment:
Use `.format` to apply arguments to the message until a proper brace-message adapter is written.